CHANGELOG.md

```diff
@@ -6,14 +6,18 @@
 - Default configurations for the deprecated Ropsten, Kiln, Shandong, and Astor networks have been removed from the CLI network list. These networks can currently be accessed but will require a user-provided genesis configuration. [#4896](https://github.com/hyperledger/besu/pull/4869)
 
 ### Additions and Improvements
-
+- Add new KZG precompile and option to override the trusted setup being used.
+- 
 ### Bug Fixes
 
 ## 23.1.0-beta
 
 ### Breaking Changes
 - GoQuorum-compatible privacy is deprecated and will be removed in 23.4
 - IBFT 1.0 is deprecated and will be removed in 23.4
+
+
+## 22.10.4
```

Comment:
another bad merge?

Reply:
indeed. converting back to draft till I can sort this tomorrow.